WIP: Setup website

trabaldini opened the pull request:
Used the color of the logo for background and links, but maybe it's a bit too much?
Used the content of the README, but it's not so much. Is there other content I could/should add?

marcoow commented on index.html:
```diff
@@ -1 +1,402 @@
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+      ga('create', 'UA-53237918-6', 'auto');
```
I just created the correct analytics ID: `UA-53237918-22`.

marcoow commented on index.html:
```diff
@@ -1 +1,402 @@
+        </div>
+      </div>
+    </div>
+    <script src="https://yandex.st/highlightjs/7.4/highlight.min.js"></script>
```
I don't think this is needed.

marcoow commented on index.html:
```diff
@@ -1 +1,402 @@
+      ga('send', 'pageview');
+    </script>
+  </body>
+</html>
```
👀

marcoow commented on index.html:
```diff
@@ -1 +1,402 @@
+      }
+    
+      a, a:visited {
+        color: #99FFCC;
```
the link text is a bit hard to read in this color - we might have to add a background color, e.g. black.

marcoow commented on index.html:
```diff
@@ -1 +1,402 @@
+      }
+
+      hr {
+        border-bottom: 1px solid #99FFCC;
```
I think this might look better in black.

trabaldini commented:
Changed the colors and removed the `.DS_Store`, but I can't find the `.travis.yml`
There is just one on the master branch but it's not the one you mean?

marcoow commented:
Hm, I assumed the `.travis.yml` was still present and thus Travis tried to build. I disabled the build for this branch in the Travis settings now.
